src/sass/components/_cards.scss

```diff
@@ -0,0 +1,14 @@
+// Product Cards
+// =============
+
+.md-product-meta {
+  row-gap: $spacing-xx-small;
+}
+
+// make version numbers font sizing match pills so card info lines up
+// whether products have tags or not
+.md-product-meta-title {
+  border: 1px solid transparent;
+  font-size: $font-size-3;
+  line-height: 1.5;
```

Comment:
In the mockup (and functionally before this PR), the version title is smaller font. This is now overriding what the `.slds-text-title` class is intended for -- do we want to apply that class to tags as well? Or get rid of it? I'm not sure what we're gaining here, and I'm hesitant to override the SLDS styles without a strong reason (even to match the mockup).

Reply:
I removed all of this in the last round of edits.